packages/.eslintrc.json

```diff
@@ -102,7 +104,15 @@
         {
             "files": ["./plugins/**/*"],
             "rules": {
-                "@typescript-eslint/no-explicit-any": "error"
+                "@typescript-eslint/no-explicit-any": "error",
+                "@dimensiondev/no-number-constructor": "error",
+                "@dimensiondev/type/no-empty-literal": [
+                    "off",
+                    {
+                        "array": "Use `EMPTY_LIST` instead of `[]`",
+                        "object": "Use `EMPTY_OBJECT` instead of `{}`"
+                    }
+                ]
```

Comment:
plz design a file match pattern on the eslint rule

Reply:
@guanbinrui @Jack-Works

Reply:
have no idea

Reply:
I think we can close this, and start to write some `best practice`.